test/Tests/src/Data/Polyglot_Spec.enso

```diff
@@ -19,9 +21,26 @@ spec = Test.group "Polyglot" <|
         Polyglot.new String ["42"] . should_equal "42"
         Polyglot.new String ["42"].to_array . should_equal "42"
 
+    Test.specify "use Integer created by constructor" <|
+        Polyglot.new Integer [42] . should_equal 42
+
+    Test.specify "use Integer read from Polyglot object" <|
+        (Polyglot.get_member js_meaning "meaning") . should_equal 42
+
+    Test.specify "use Integer obtained from a call" <|
+        Integer.parseInt "42" . should_equal 42
+
+    Test.specify "use Integer obtained from a read" <|
+        hold = IntHolder.new (6 * 7)
+        hold.value . should_equal 42
+
     Test.specify "should be able to execute a polyglot function object along with corresponding arguments" <|
         fun = Function.identity
         Polyglot.execute fun ["42"] . should_equal "42"
         Polyglot.execute fun ["42"].to_array . should_equal "42"
 
 main = Test_Suite.run_main spec
+
+
+foreign js js_meaning = """
+    return { meaning : 6 * 7 };
```

Comment:
can we keep "main" at the bottom please

Reply:
The textmate grammar that we currently use has problems with `"""` - then everything below is treated as a string including the `main` function. That's why I have a habit to put `foreign` functions to the bottom.

However you are right, we should fix the grammar.

Reply:
Fixed in f493c6d3f